dev/com.ibm.ws.wssecurity.3.4.1/src/com/ibm/ws/wssecurity/internal/WSSecurityConstants.java

```diff
@@ -88,5 +89,5 @@ public class WSSecurityConstants {
     public static final String KEY_timeToLive = "timeToLive";
     public static final String KEY_audienceRestrictions = "audienceRestrictions";
 
-    public static final String WSSEC_DEFAULT_SIGNATURE_ALGORITHM = CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA1;
+    public static final String WSSEC_DEFAULT_SIGNATURE_ALGORITHM = CryptoUtils.isFips140_3EnabledWithBetaGuard() ? CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA256 : CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA1;
 }
```

Comment:
Used in WSSecurityClientConfiguration / WSSecurityConfiguration if the property `signatureAlgorithm` isn't set this default is used.

Reply:
Not setting globally because this default would need discussion, good discussion for the team to understand how/where this signature is used in web services.